jvb/src/main/resources/reference.conf

```diff
@@ -153,8 +153,8 @@ videobridge {
     resolve-remote-candidates = false
 
     # The nomination strategy to use for ICE.  THe accepted values are defined in ice4j's NominationStrategy:
-    # "NominateFirstValid", "NominateHighestPriority", "NominateFirstHostOrReflexiveValid", or "NominateBestRTT".
-    nomination-strategy = "NominateFirstValid"
+    # "NOMINATE_FIRST_VALID", "NOMINATE_HIGHEST_PRIORITY", "NOMINATE_FIRST_HOST_OR_REFLEXIVE_VALID", or "NOMINATE_BEST_RTT".
+    nomination-strategy = "NOMINATE_FIRST_VALID"
```

Comment:
Hm?  Don't these strings come from the `NominationStrategy` enum in ice4j?  What's converting this from CamelCase to ALL_CAPS?

Reply:
The reason is that the old code retrieved as a `String` and used a `fromString` helper.  In the new code I can retrieve the enum type directly, but that uses the built-in `enumValueOf` which expected the value to match the enum _name_ itself, which is in all caps.

Reply:
Hm.  The problem with that is that it breaks old config files, of course (including ours).  How hard would it be to make it use the `fromString` helper instead?

Reply:
Well...lucky for us I happen to know a lib which can accommodate exactly such a breaking change!  (Though I don't know why I didn't think to address that).  I can configure the property like so:

```
val nominationStrategy: NominationStrategy by config {
    retrieve("videobridge.ice.nomination-strategy".from(JitsiConfig.newConfig))
    retrieve("videobridge.ice.nomination-strategy".from(JitsiConfig.newConfig).asType<String>().andCovertyBy(NominationStrategy::FromString))
}
```

Will add a test to verify that works as expected.

Reply:
(And same for `keep-alive-strategy`)

Reply:
I'd be inclined to think we just need the `andType`...`andConvertBy` flavor, but if you want both, that's fine.

Doesn't the converted one need to be first, because the unconverted one will pull from the defaults file?

Reply:
The merge of the defaults file and the overrides file is already done before this, so both of them are just trying to parse the 'final' config value in their own way.  

I did have to move the "parse as enum" one afterwards, though, because I give a rich error message when an enum value is bad that I bubble all the way up (i.e. it doesn't fall-through to another supplier) to help catch typos, etc, so if it was the `NominateFirstValid` style, then we'd throw an exception rather than fall through.

However, after changing the order, I encountered a strange situation (kotlin bug?) where, even though I have the type specified as a non-nullable one, the `KeepAliveStrategy` value, because it's a platform type coming from Java, allows 'null' to bubble all the way up, meaning if it's a new-style value (e.g. `NOMINATE_FIRST_VALID`), 
```kotlin
retrieve("videobridge.ice.nomination-strategy".from(JitsiConfig.newConfig).asType<String>().andCovertyBy(NominationStrategy::FromString))
```
will successfully return a value of `null` for the property (won't fall through).  I was relying on this throwing an exception, but weirdly it doesn't.  I think I'll revert back to the old style (parse as `String`, use `fromString`) only for this and track a bug in jitsi-metaconfig to look into the other issue further.